scheduler/bin/gcloud-make-for-integration.sh

```diff
@@ -0,0 +1,56 @@
+##
+## Configure a kubernetes cluster for running pool-based integration tests and running pools in general.
+##
+##
+
+
+set -e
+
+## Example arguments that work:
+##
+
+#ZONE=us-central1-a
+#CLUSTERNAME=test-cluster-2
+#PROJECT=<FILL SOMETHING IN>
+
+PROJECT=$1
+ZONE=$2
+CLUSTERNAME=$3
+
+echo "---- Building kubernetes cluster for project=$PROJECT zone=$ZONE clustername=$CLUSTERNAME"
+
+# Nuke all existing temporary clusters; don't want to keep on making more idle clusters each time you invoke this.
+echo "---- Deleting any existing temporary clusters."
+echo "---- Will delete the following. If you don't want them gone, control-c now"
+gcloud container clusters list --filter 'resourceLabels.longevity=temporary'
+sleep 10
+for i in `gcloud container clusters list --filter 'resourceLabels.longevity=temporary' --format="value(name)"` ; do gcloud --quiet container clusters delete $i ; done
+
+# This creates a cluster, with some nodes that are needed for kubernetes management.
+# Create 7 nodes, with three tainted for alpha, 3 untainted and 3 tainted with gamma pool.
```

Comment:
Why are 3 untainted?

Reply:
For GKE's own uses for its system pods. Also convenient to have around if I'm doing non-pool tests.

Reply:
Can we add that info to the comment?

Reply:
@scrosby Thoughts on this?

Reply:
Done.